proc_IR_noecho.py

```diff
@@ -53,7 +56,8 @@
             label='vd=%g'%s.getaxis('indirect')[j])
     #}}}
     #{{{exponential curve with fit
-    rec_curve = s['t2':(0,None)].sum('t2')
+    s.ft('t2')
+    rec_curve = s['t2':(-150,150)].sum('t2')
```

Comment:
the point before was to integrate in the frequency domain -- you can't sum the FID, which is what I think you were trying to do

Reply:
thumbs up or something if this makes sense